doc/COMPILING/COMPILING-VS-VCPKG.md

```diff
@@ -50,6 +54,12 @@ cd vcpkg
 .\vcpkg upgrade
 ```
 
+#### add libs for Visual Studio compiler:
+
+```cmd
+.\vcpkg integrate install
+```
+
```

Comment:
```suggestion
```

***

This command is already mentioned above:

![image](https://user-images.githubusercontent.com/16213433/99040660-916d9000-259a-11eb-8e77-9ddc9d170077.png)

Reply:
vcpkg command `.\vcpkg integrate install` installs its own libraries for visual studio. But after `.\bootsrap-vcpkg.bat -disableMetrics` no additional local libraries (dynamic-load,libflac,mpg123,libmodplug,libvorbis etc.) are available. In this case аdditional libraries are installed by the Visual studio itself. But it can fail the installation. Therefore we can install additional packages ourselves. But they need to be linked to the VS manually. See https://docs.microsoft.com/ru-ru/cpp/build/vcpkg?view=msvc-160

Reply:
Simply after each command `.\vcpkg instal`
must follow next command `.\vcpkg integrate install`

Reply:
Integration should be done only once after vcpkg installation and should work for all existing and newly installed packages (https://vcpkg.readthedocs.io/en/latest/users/integration/#integrate-command). If it does not work - something is broken in vcpkg.

Reply:
Ideally build should succeed with just these following steps:

![image](https://user-images.githubusercontent.com/16213433/99046643-1c06bd00-25a4-11eb-8d97-79b953fb9686.png)

All packages should be installed automatically using `vcpkg.json` manifest and no prior installation should be required required. It worked like that previously when I implemented this in https://github.com/CleverRaven/Cataclysm-DDA/pull/41908.

It does not work now because of `mpg123` failing to build, so that is the issue with that one port or vcpkg itself.

Try using clean vcpkg installation (don't forget to remove cached binaries from `%LOCALAPPDATA%\vcpkg`) with integration enabled and packages installed - build will fail (because of `mpg123`):

But when you remove `mpg123` from manifest - build will succeed:

![image](https://user-images.githubusercontent.com/16213433/99047061-b109b600-25a4-11eb-940d-fa9ef8a1a1f6.png)

Reply:
> If it does not work - something is broken in vcpkg.

Ok. I hope nothing is broken in vpkg.